jdisc_core/src/main/java/com/yahoo/jdisc/application/OsgiFramework.java

```diff
@@ -60,6 +60,8 @@
     /**
      * Synchronously refresh all bundles currently loaded. Once this method returns, the
      * class loaders of all bundles will reflect on the current set of loaded bundles.
+     *
+     * NOTE: This method is no longer used by the Jdisc container framework, but kept for completeness.
```

Comment:
Mark as deprecated?

Reply:
I originally did, but opted against it. It's really a natural part of the framework, and hence just a comment that it's not used by the jdisc container. It is extensively used in unit/integration tests.